### PR TITLE
SW-1860 Nurseries with Show More in Inventory Table 

### PR DESCRIPTION
**Pull request by charlie-tf**
- import and use the TextTruncated component

image
image
image
image

### CODE REVIEW

**Comment by karthikbtf** [comment]
vercel didn't deploy here

**Comment by karthikbtf** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Updated
> **tree-location-web**	🔄 Building ([Inspect](https://vercel.com/terraformation/tree-location-web/6X8ik4K6Bk1h6jPG8zVMtYmR6AZk))		Oct 27, 2022 at 8:39PM (UTC)

finally

File: src/components/Inventory/InventoryCellRenderer.tsx [hunk 1]
```diff
@@ -17,11 +18,14 @@ export default function InventoryCellRenderer(props: RendererProps<TableRowType>
...
-    nurseries.forEach((n) => {
-      nurseriesNames += n.facility_name;
-    });
-    return nurseriesNames;
+    return (
+      <TextTruncated
+        stringList={nurseries.map((n) => n.facility_name)}
+        maxLengthPx={250}
+        textStyle={{ fontSize: 14 }}
+        showAllStyle={{ padding: '17px', fontSize: 16 }}
```
Comment (karthikbtf) [inline src/components/Inventory/InventoryCellRenderer.tsx:26]: possible to use theme.spacing() here? we can get 16px with theme.spacing(2)

File: src/components/Inventory/InventoryCellRenderer.tsx [hunk 2]
```diff
@@ -17,11 +18,14 @@ export default function InventoryCellRenderer(props: RendererProps<TableRowType>
...
   const getNurseriesNames = (nurseries: any[]) => {
-    let nurseriesNames = '';
-    nurseries.forEach((n) => {
-      nurseriesNames += n.facility_name;
-    });
-    return nurseriesNames;
+    return (
+      <TextTruncated
+        stringList={nurseries.map((n) => n.facility_name)}
+        maxLengthPx={250}
```
Comment (karthikbtf) [inline src/components/Inventory/InventoryCellRenderer.tsx:24]: we could keep an eye on this and see if we need to provide some calculated width (some % of total width)?

**Review by karthikbtf — APPROVED** [review]
nice 🎉

**Comment by charlie-tf** [comment]
hmm, i noticed my font is smaller than the rest of the table fonts. I used 14px font per the [figma](https://www.figma.com/file/GuJDioJNGC91EF10YVXuGD/Nursery-Management?node-id=1012%3A595595), looks like the other text here is 16px. @constanzauanini or @karthikbtf should I update mine to 16px?
image

**Comment by karthikbtf** [comment]
> hmm, i noticed my font is smaller than the rest of the table fonts. I used 14px font per the [figma](https://www.figma.com/file/GuJDioJNGC91EF10YVXuGD/Nursery-Management?node-id=1012%3A595595), looks like the other text here is 16px. @constanzauanini or @karthikbtf should I update mine to 16px? image

In figma they are all 14px, 16px is probably incorrect and needs to be fixed

**Comment by charlie-tf** [comment]
>In figma they are all 14px, 16px is probably incorrect and needs to be fixed

Okay, I'll leave mine at 14px 👍

**Comment by karthikbtf** [comment]
> > In figma they are all 14px, 16px is probably incorrect and needs to be fixed
> 
> Okay, I'll leave mine at 14px 👍

Thanks @charlie-tf can you open a bug under the nursery epic (clean up tasks story)?

File: src/components/Inventory/InventoryCellRenderer.tsx (hunk 2 above)
Comment (charlie-tf) [reply src/components/Inventory/InventoryCellRenderer.tsx:24]: I'll move this up into a constant at least so that it's more visible

**Comment by constanzauanini** [comment]
> > In figma they are all 14px, 16px is probably incorrect and needs to be fixed
> 
> Okay, I'll leave mine at 14px 👍

Sorry, I may have missed that

**Comment by karthikbtf** [comment]
> > > In figma they are all 14px, 16px is probably incorrect and needs to be fixed
> > 
> > 
> > Okay, I'll leave mine at 14px 👍
> 
> Sorry, I may have missed that

It's all good, we can keep track of cleanup items here https://terraformation.atlassian.net/browse/SW-2032

**Comment by charlie-tf** [comment]
> > > In figma they are all 14px, 16px is probably incorrect and needs to be fixed
> > 
> > 
> > Okay, I'll leave mine at 14px 👍
> 
> Sorry, I may have missed that

No worries! I created a ticket: https://terraformation.atlassian.net/browse/SW-2062

**Pull request merged**